frontend/src/components/EventCard.tsx

```diff
@@ -1,28 +1,46 @@
 import React from 'react';
+import {Card} from "react-bootstrap";
 import {EventModel} from "../model/EventModel";
-import './EventCard.css';
-import moment from "moment";
+import DateComponent from "./DateComponent";
+import CategoryIcon from "./CategoryIcon";
+import "./EventCard.css"
 
-type Props={
-    event:EventModel
+type Props = {
+    event: EventModel
 }
-function EventCard(props:Props) {
+
+function EventCard(props: Props) {
 
     function renderEventImage(event: EventModel) {
         if (event.image) {
-            return <img src={`data:image/jpeg;base64,${event.image}`} alt={event.title}/>;
+            return `data:image/jpeg;base64,${event.image}`;
         } else {
-            return <img src={event.imageUrl} alt={event.title}/>;
+            return event.imageUrl;
         }
     }
 
     return (
-        <div className="event-card">
-            {renderEventImage(props.event)}
-            <h4>{props.event.title}</h4>
-            <h5>{moment(props.event.start).format('DD.MM.YYYY HH:mm:ss [Uhr]')}</h5>
-            <h5><a href={props.event.source}>{props.event.location}</a></h5>
-        </div>
+        <Card style={{width: '18rem'}} bg="light">
+            <Card.Header className="d-flex align-items-center justify-content-center text-center">
+                <CategoryIcon category={props.event.category}/>&nbsp;<DateComponent date={props.event.start}/>
+            </Card.Header>
+            <Card.Img variant="top" src={renderEventImage(props.event)} className="card-img-top"/>
+            <Card.Body>
+                <Card.Title>{props.event.title}</Card.Title>
+                <Card.Subtitle className="text-truncate">{props.event.location}</Card.Subtitle>
+                <Card.Text className="text-truncate">
+                    {props.event.description}
+                </Card.Text>
+            </Card.Body>
+            {/*<ListGroup className="list-group-flush">
+                <ListGroup.Item>{props.event.location}</ListGroup.Item>
+            </ListGroup>*/}
```

Comment:
Warum ist das hier ausgeklammert?

Reply:
as a reminder for integrating the list group construct in a further pull request